perun-wui-core/src/main/java/cz/metacentrum/perun/wui/json/ErrorTranslator.java

```diff
@@ -49,7 +49,7 @@ public static String getTranslatedMessage(PerunException error) {
 			GeneralObject holder = error.getAttributeHolder();
 			GeneralObject secondHolder = error.getAttributeHolderSecondary();
 
-			String text = "Wrong value of attribute (value or format).<p>";
+			String text = error.getMessage();
```

Comment:
Our error messages are not generally human readable, so I suggest not to change this to simply return message of the exception. If you need to show specific text for ssh keys I suggest to implement check on attribute name and return message only for such attribute.